src/Aspirate.Services/Extensions/JsonExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System.Text.Json;
+
+namespace Aspirate.Services.Extensions;
+public static class JsonExtensions
+{
+    public static JsonDocument TryParseJson(this string json)
```

Comment:
My bad - lets make this a nullable return too `public static JsonDocument? TryParseJson(this string json)`